src/pepr/uds-operator-config/values.yaml

```diff
@@ -2,18 +2,35 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
 
 operator:
+  ### DEPRECATED: This section is deprecated and will be removed in a future release. ###
   # Domain configuration (admin defaults to `admin.UDS_DOMAIN`)
   UDS_DOMAIN: "###ZARF_VAR_DOMAIN###"
   UDS_ADMIN_DOMAIN: "###ZARF_VAR_ADMIN_DOMAIN###"
   UDS_CA_CERT: "###ZARF_VAR_CA_CERT###"
   UDS_ALLOW_ALL_NS_EXEMPTIONS: "###ZARF_VAR_ALLOW_ALL_NS_EXEMPTIONS###"
   UDS_LOG_LEVEL: "###ZARF_VAR_UDS_LOG_LEVEL###"
-  AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   KUBEAPI_CIDR: ""
   KUBENODE_CIDRS: ""
+  ### END DEPRECATED ###
+  AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   # Allow Pepr watch to be configurable to react to dropped connections faster
   PEPR_LAST_SEEN_LIMIT_SECONDS: "300"
   # Allow Pepr to re-list resources more frequently to avoid missing resources
   PEPR_RELIST_INTERVAL_SECONDS: "600"
   # Configure Pepr reconcile strategy to have separate queues for faster reconciliation
   PEPR_RECONCILE_STRATEGY: "kindNsName"
+
+cluster:
+  attributes:
+    UDS_CLUSTER_NAME: "###ZARF_VAR_CLUSTER_NAME###"
+    UDS_CLUSTER_TAGS: "###ZARF_VAR_CLUSTER_TAGS###"
```

Comment:
For cluster name and tags imo we don't need to expose a zarf var for these, unless there's a specific UI use case I'm missing here - they could just be set as overrides?

Specific to tags, shouldn't that be a list rather than a string (maybe this is string because of zarf var right now though)? It could also be useful to have a values schema for this chart now that we're adding more complexity to it.

Reply:
On the tag comment I think that would also help clean up the helm template for tags if switched to a proper list.